README.md

```
@@ -73,6 +73,7 @@ These demonstrate SFILES 2.0 strings for a variety of PFDs and P\&IDs.
 | <img src="docs/photos/Gabriel.jpg" width="50"> | [Gabriel Vogel](https://www.pi-research.org/author/gabriel-vogel/) | <a href="https://www.linkedin.com/in/gabriel-vogel-2512a9212/" rel="nofollow noreferrer"> <img src="https://i.sstatic.net/gVE0j.png" >  </a> <a href="https://scholar.google.com/citations?user=9yRGFZMAAAAJ&hl=en" rel="nofollow noreferrer"> <img src="docs/logos/google-scholar-square.svg" width="14">  </a> |
 |  | [Edwin Hirtreiter](https://www.pi-research.org/author/edwin-johann-hirtreiter/) | <a href="https://www.linkedin.com/in/edwin-hirtreiter-4ab3481a5/" rel="nofollow noreferrer"> <img src="https://i.sstatic.net/gVE0j.png" >  </a> <a href="https://scholar.google.com/citations?hl=en&user=QURe_SoAAAAJ" rel="nofollow noreferrer"> <img src="docs/logos/google-scholar-square.svg" width="14">  </a> |
 | <img src="docs/photos/Lukas.jpg" width="50"> | [Lukas Schulze Balhorn](https://www.pi-research.org/author/lukas-schulze-balhorn/) | <a href="https://www.linkedin.com/in/lukas-schulze-balhorn-12a3a4205/" rel="nofollow noreferrer"> <img src="https://i.sstatic.net/gVE0j.png" >  </a> <a href="https://scholar.google.com/citations?user=LZZ7piQAAAAJ&hl=en" rel="nofollow noreferrer"> <img src="docs/logos/google-scholar-square.svg" width="14">  </a> |
+|  | Antonio Rocha Azevedo | <a href="https://www.linkedin.com/in/antonio-rocha-azevedo-9577b41a1/" rel="nofollow noreferrer"> <img src="https://i.sstatic.net/gVE0j.png" >  </a> |
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)